test/suites/drift/suite_test.go

```diff
@@ -148,20 +148,83 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(node), &v1.Node{})).To(Succeed())
 		}).WithTimeout(time.Minute).Should(Succeed())
 	})
-})
+	It("should deprovision nodes that have drifted due to securitygroup", func() {
```

Comment:
Can you consider using `By()` statements? We've started using those more and that gives us some indication of the step that is occurring at a given time for free from Ginkgo